netaddr/tests/ip/test_ip_v4.py

```diff
@@ -487,3 +487,8 @@ def test_spanning_cidr_handles_strings():
     ]
     assert spanning_cidr(addresses) == IPNetwork('10.0.0.0/29')
     assert spanning_cidr(reversed(addresses)) == IPNetwork('10.0.0.0/29')
+
+
+def test_ipnetwork_nohost():
+    assert IPNetwork('192.168.0.1/24', flags=NOHOST).ip == IPAddress('192.168.0.0')
```

Comment:
Can you please remove this line? Yes, it's a test worth having but it's a test out of scope of this PR (which is about fixing the copy constructor).

Reply:
I have removed the out of scope test